Userland/Shell/main.cpp

```diff
@@ -181,6 +181,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     parser.add_option(posix_mode, "Behave like a POSIX-compatible shell", "posix");
     parser.add_positional_argument(file_to_read_from, "File to read commands from", "file", Core::ArgsParser::Required::No);
     parser.add_positional_argument(script_args, "Extra arguments to pass to the script (via $* and co)", "argument", Core::ArgsParser::Required::No);
+    parser.add_ignored("login", 'l');
```

Comment:
Unrelated, put in separate commit?

Reply:
Oh I don't actually want that in there, will remove.